### DISCUSSION TITLE
problem with lxc and jellyfin, "almost" solved, pls help

### DISCUSSION THREAD

**Discussion by rcv11x** [tteck/Proxmox#1068] [Help Needed]
Guys I have a question to see if you can help me, I have jellyfin inside a proxmox LXC, the transcoding was going well until a few days ago I updated proxmox, I get an error now after being several hours I have realized that I get an error in the log of jellyfin:

`[AVHWDeviceContext @ 0x556b9f5a66c0] No VA display found for device /dev/dri/renderD128.
Device creation failed: -22.
Failed to set value 'vaapi=va:/dev/dri/renderD128' for option 'init_hw_device': Invalid argument
Error parsing global options: Invalid argument`

it turns out that if inside the lxc I run:

`ls -l /dev/dri/`

I get back:

`crw-rw---- 1 root syslog 226, 128 Feb 15 07:37 renderD128`

where it says syslog should be render, if I change the permissions with: 

`sudo chown root:render /dev/dri/renderD128`

magically the transcoding works and no longer gives me error! but ... when I restart the pc proxmox pc that permission is set again in syslog, how could I fix this? I do not know if this happened before because I did not look but it should be render that is how it works not with "syslog" if someone can help me I would appreciate it very much, I think it has an easy solution

**Comment by RealJustMe** [comment]
i had this problem with emby an my solution was this:
https://github.com/tteck/Proxmox/discussions/747#discussioncomment-4558545

**Reply by rcv11x** [reply]
hey, thanks a lot for your answer! when i get home later i'll try it out ;)

**Reply by RealJustMe** [reply]
Hope it works for you too :) 
Just be sure to get the correct group ID for render (107 in my case)

**Reply by Stooovie** [reply]
Doesn't fix my issue (I have the same original error message: [AVHWDeviceContext @ 0x556b9f5a66c0] No VA display found for device /dev/dri/renderD128. Device creation failed: -22. Failed to set value 'vaapi=va:/dev/dri/renderD128' for option 'init_hw_device': Invalid argument Error parsing global options: Invalid argument)

`ls -l /dev/dri/` returns
```
total 0
-rw-rw-rw- 1 root render 0 Nov  6 17:10 renderD128
```
and `sudo chown root:render /dev/dri/renderD128` doesn't fix it. It has been working for 6 months until today.
